### ISSUE TITLE
Popups are too small

### ISSUE THREAD

**Issue by JeroenDeDauw** [JeroenDeDauw/Maps#490]
> #Display_map marker popup have a much smaller space then what the old version allowed. The old version allowed external lines. Now the field is much shorter and less useful.

-- @acnetj in https://github.com/JeroenDeDauw/Maps/issues/488

**Comment by JeroenDeDauw** [comment]
@acnetj some extra info here would be great. Do you have screenshots of both the desired and undesired behavior? How exactly should the behavior that is in the last release be changed?

**Comment by acnetj** [comment]
Desired infowindow: 
![image](https://user-images.githubusercontent.com/10605369/48605057-8db5bd00-e930-11e8-805a-304eb2d3b362.png)

Code that produced it is here: https://www.transit.wiki/index.php?title=Template:GTFS_map&action=edit&oldid=174191

This code doesn't run due to last ";" bug, but even if I hand edit it the marker won't show unless I reduced the text in infowindow.

**Comment by JeroenDeDauw** [comment]
What does the "undsired infowindow" look like? And what napping service (ie Google Maps, Leaflet) are you seeing this issue with?

**Comment by acnetj** [comment]
Actually upon further examination. I see that it is infowindow with materials that are or look like query strings that would not be rendered, whether it is a formal link or not (anything that has something like ?a= or &b=)

https://www.transit.wiki/Sandbox_2#Infowindow

**Comment by JeroenDeDauw** [comment]
Can you rephrase that? I don't understand.

Also, please keep in mind that if you link to a page you keep changing, the link ends up being more confusing then helpful when I view the issue a week later.

**Comment by acnetj** [comment]
Basically if the infowindow has any text that resembles a query string (real or not) nothing will be rendered (no marker, no infowindow, just a blank map).

Anything that looks like a query string (?a=b or &a=b) in XXX,YYY, and ZZZ will cause the map to fail.

{{#display_map: 37.775110, -122.419320~XXX~YYY~ZZZ|height=250px}}

**Comment by JeroenDeDauw** [comment]
Well yeah, that is because the `=` symbol is not allowed there when you do not name the parameter.

Instead of

```
{{#display_map: 37.775110, -122.419320~?a=b}}
```

you can do this

```
{{#display_map: locations=37.775110, -122.419320~?a=b}}
```

**Comment by JeroenDeDauw** [comment]
So... I still need this info to continue:

> What does the "undsired infowindow" look like? And what napping service (ie Google Maps, Leaflet) are you seeing this issue with?

**Comment by acnetj** [comment]
What you identified was the issue. So basically you have this workaround which is pretty acceptable. Other than the tailing ";" the workaround should create the map what I showed on top.

Basically the old code won't produce what I wanted under the current version, therefore it would be undesirable if I have to get ride of the url string included originally to produce a map in the current version of the extension.

**Comment by JeroenDeDauw** [comment]
How is that related to "Popups are too small"?
